src/seedcase_sprout/core/sprout_checks/check_data.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+
+import polars as pl
+
+from seedcase_sprout.core.get_nested_attr import get_nested_attr
+from seedcase_sprout.core.properties import FieldProperties, ResourceProperties
+from seedcase_sprout.core.sprout_checks.check_data_header import (
+    check_data_header,
+)
+from seedcase_sprout.core.sprout_checks.check_resource_properties import (
+    check_resource_properties,
+)
+from seedcase_sprout.core.sprout_checks.resource_properties_to_pandera_schema import (
+    resource_properties_to_pandera_schema,
+)
+
+
+def check_data(data_path: Path, resource_properties: ResourceProperties) -> Path:
+    """Checks that the CSV data file at `data_path` matches the resource properties.
+
+    Runs the following checks:
```

Comment:
```suggestion
    Runs the following checks:
    
```

To render the Markdown list correctly.